.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,54 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+    - master
+    
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+#     - uses: gaurav-nelson/github-action-markdown-link-check@0.4.0
+# TODO: This fails because we have a localhost link in the README. Consider removing or otherwise ignoring.
+  linting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+#     - name: run-linter
+#       run: |
+#         yarn install
+#         yarn lint
+#       env:
+#         CI: true
+# TODO: This fails because we don't have linting... Consider (please) adding strict linting.
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: npm install, build, and test
+      run: |
+        npm install
+        npm run build
+        npm run test
+      env:
+        CI: false # TODO: revisit this. npm build turns extra checks on when this is true and something there is broken
+    - uses: redhat-cop/github-actions/s2i@v1
+      with:
+        base: registry.access.redhat.com/rhscl/httpd-24-rhel7
```

Comment:
same here i think. We should probably pin to a version.

Reply:
👍